core/grpc/proto/kms.v1.proto

```diff
@@ -420,22 +420,22 @@ message KmsContext {
   int32 threshold = 5;
 }
 
-// The information held by the custodian nodes in a custodian context.
-message CustodianNode {
-  // The human-readable name of the custodian node.
-  string name = 1;
-  // The public encryption key of the custodian node,
-  // that the KMS parties use to back their shares to the custodian.
-  bytes public_encryption_key = 2;
+message CustodianSetupMessage {
```

Comment:
These are never used yet so there should be no danger in modifying these at this point